src/mixins/document-mixin.ts

```diff
@@ -17,25 +18,17 @@ export default class DocumentMixin extends Vue {
       validationErrorMsg: 'Document must be set to fit onto 8.5” x 11” letter-size paper.'
     }
   }
-  readonly DOCUMENT_TYPES = {
-    contInAuthorization: {
-      class: 'CORP',
-      type: 'CNTA'
-    },
-    affidavitDocument: {
-      class: 'CORP',
-      type: 'DIRECTOR_AFFIDAVIT'
-    }
-  }
 
   pdfjsLib: any
+  documentTypes: any
```

Comment:
For cleaner code, I think you can actually do `import { DOCUMENT_TYPES as documentTypes } from ....`